server/templates/donate-form.html

```diff
@@ -57,17 +59,29 @@ <h2 class="grid_separator link--teal">Become a Texas Tribune Member</h2>
             </div>
           </div>
         </div>
+        {% if not use_thermometer %}
         <div class="col_5">
           {% include 'includes/faq.html' %}
         </div>
+        {% endif %}
       </div>
     </section>
 
     <section class="section-padding grid_padded--xl section-padding--filled">
-      <div class="member_benefits grid_container">
-        {% include 'includes/benefits.html' %}
-      </div>
-
+      {% if use_thermometer %}
+        <div class="grid_container grid_row grid_wrap--l gap-s l-swap--from-bp-m">
+          <div class="col_5">
+            {% include 'includes/faq.html' %}
+          </div>
+          <div class="col_7">
+            {% include 'includes/benefits.html' %}
+          </div>
+        </div>
+        {% else %}
+          <div class="member_benefits grid_container">
+            {% include 'includes/benefits.html' %}
+          </div>
```

Comment:
**Nitpick (non-blocking):** annoying again, but can you fix this indentation?

Reply:
Fixed in [5fd0d92](https://github.com/texastribune/donations/pull/1231/commits/5fd0d927f22755b9e70d24cd5916b0b7ae2ff404), good catch!